modules/benchmark/R/format_wide2long.R

```diff
@@ -0,0 +1,103 @@
+##' @name format_wide2long
+##' @title format_wide2long
+##' 
+##' @param out wide format data
+##' @param format as returned by query.format.vars
+##' @param vars_used 
+##' @param time.row
```

Comment:
it is, and then it is extracted in `load_data` before things are passed to this function, I could update `format` within this function and re-extract `time.row` in `load_data` once the output of this function is returned, but I decided to handle those here to keep `load_data` a bit cleaner